### PR TITLE
implemented normalization

### PR DESCRIPTION
**Pull request by dvd42**

### CODE REVIEW

File: torchgeometry/augmentation/normalization.py [hunk 1]
```diff
@@ -1,24 +1,71 @@
...
+        inplace (bool): Perform normalisation in-place.
+
+    Returns:
+        Tensor: The normalised image tensor.
+    """
+
+    assert len(mean) == data.shape[-3], 'Mean lenght and number of channels do not match'
+    assert len(std) == data.shape[-3], 'Std lenght and number of channels do not match'
+
+    if not isinstance(data, torch.Tensor):
```
Comment (edgarriba) [inline torchgeometry/augmentation/normalization.py]: use torch.is_tensor

File: torchgeometry/augmentation/normalization.py [hunk 2]
```diff
@@ -1,24 +1,71 @@
...
+        data (Tensor): The image tensor to be normalised.
+        mean (Sequence of float): Mean for each channel.
+        std (Sequence of float): Standard deviations for each channel.
+        inplace (bool): Perform normalisation in-place.
+
+    Returns:
+        Tensor: The normalised image tensor.
+    """
+
+    assert len(mean) == data.shape[-3], 'Mean lenght and number of channels do not match'
```
Comment (edgarriba) [inline torchgeometry/augmentation/normalization.py]: use raise ValueError

File: torchgeometry/augmentation/normalization.py [hunk 3]
```diff
@@ -1,24 +1,71 @@
...
     def __repr__(self):
-        return self.__class__.__name__ + '()'
+        repr = '(mean={0}, std={1}, inplace={2})'.format(self.mean,
+                                                         self.std, self.inplace)
+        return self.__class__.__name__ + repr
+
+
+def normalise(data: torch.Tensor, mean: Sequence[float],
+              std: Sequence[float],
+              inplace: Optional[bool] = False) -> torch.Tensor:
```
Comment (edgarriba) [inline torchgeometry/augmentation/normalization.py]: assume arguments are tensors

File: test/test_augmentation.py [hunk 4]
```diff
@@ -103,6 +103,55 @@ def test_gradcheck(self):
...
+        # expected output
+        expected = torch.tensor([1.25, 1., 0.5]).repeat(2, 1, 1).view_as(data)
+
+        f = taug.Normalize(mean, std)
+        assert_allclose(f(data), expected)
+
+    def test_gradcheck(self):
+
+        # prepare input data
+        data = torch.ones(2, 3, 1, 1).double()
```
Comment (edgarriba) [inline test/test_augmentation.py]: ```.double()``` not needed since it's done inside tensor_to_gradcheck_var()

**Comment by edgarriba** [comment]
@dvd42 as we discussed can you send this PR to master and include the functionality into the image module

**Comment by edgarriba** [comment]
closing in favor of https://github.com/arraiyopensource/torchgeometry/pull/117
